README.md

```diff
@@ -32,6 +32,18 @@ To build an Amazon EKS Worker AMI for a particular Kubernetes version run the fo
 ```bash
 make 1.23 ## Build a Amazon EKS Worker AMI for k8s 1.23
 ```
+
+### AMI template variables
+
+Default values for most variables are defined in [a default variable file](eks-worker-al2-variables.json).
+
+Users have the following options for specifying their own values:
+
+1. Provide a variable file with the `PACKER_VARIABLE_FILE` argument to `make`. Values in this file will override values in the default variable file. Your variable file does not need to include all possible variables, as it will be merged with the default variable file.
```

Comment:
Would be possible to write tests for these behaviors?

Reply:
Nevermind, I saw your comment stating that packer is the one which does this so no point in testing that. Contents of that comment might be good for this section too. :)

Reply:
I'd err on the side of fewer impl details in this doc, but I don't feel too strongly. Can totally add if you think it's necessary